### PR TITLE
fix: add qtlottie to linux build

### PR DESCRIPTION
**Pull request by richard-ramos**
Once merged, the docker image should be updated

### CODE REVIEW

**Comment by richard-ramos** [comment]
Proof of functionality:

image

**Review by alaibe — APPROVED** [review]

File: build-linux.sh [hunk 1]
```diff
@@ -13,5 +13,5 @@ docker run -it --rm \
   -u jenkins:$(getent group $(whoami) | cut -d: -f3) \
   -v "${PWD}:/status-desktop" \
   -w /status-desktop \
-  statusteam/nim-status-client-build:1.0.2 \
```
Comment (jakubgs) [inline build-linux.sh:16]: That's not gonna fly, as I don't update the `latest` flag normally. Better to keep it up to date.
And the current tag is [1.1.3](https://hub.docker.com/layers/statusteam/nim-status-client-build/1.1.3/images/sha256-e345051d26e5a1775ba78a395516f123781cacf2b32d0d692bdf42a11f60990f?context=explore).

**Comment by jakubgs** [comment]
>Once merged, the docker image should be updated

Incorrect. There is no automation that builds this image, nor any automation that updates the tags used in `Jenkinsfile`:
https://github.com/status-im/status-desktop/blob/c80e80bc31584b8d9977a6bb03f9c2a7cea0181d/ci/Jenkinsfile.linux#L7

**Comment by jakubgs** [comment]
I've built and pushed the image under [`statusteam/nim-status-client-build:1.1.4`](https://hub.docker.com/layers/statusteam/nim-status-client-build/1.1.4/images/sha256-3318934563fdd6d1092fa6c3964a3087ec61e20ee629f6d628b7a7291d21b590?context=explore) tag.

**Review by jakubgs — APPROVED** [review]
That looks good.

**Pull request merged**